### ISSUE TITLE
Use grabbit to get the arranged data structure

### ISSUE THREAD

**Issue by oesteban** [poldracklab/preprocessing-workflow#102]
With grabbit we aim at having the following dictionary from each subject:

```
{
  'sub-XXXX': {
      'func': [
         {
             'bold': '<path>/<prefix>ses-01_task-01_acq-01_run-01_bold.nii.gz',
             'sbref': '<path>/<prefix>ses-01_task-01_acq-01_run-01_sbref.nii.gz'
         },
        { ... as many 'bold' & 'sbref' pairs as sessions x tasks x acqs x runs }
      ],
      'anat': [
        {
            't1w': '<path>/<prefix>ses-01_run01_T1w.nii.gz',
            't2w': '<path>/<prefix>ses-01_run01_T2w.nii.gz',
        },
        { ... as many 't1w' volumes or 't1w' & 't2w' pairs as sessions x acqs x runs }
     ],
     'fmap': [ '<path>/<prefix>ses-01_run01_phasediff.nii.gz', ... (all fielmap types and sessions)]
  }
}
```

The rationale behind this structure:
  - The anatomical images are processed only once per subject.
  - The fieldmaps are processed only once per subject (meaning, we don't implement the ```IntendedFor```).
  - One functional workflow per functional image in the func folder, i.e. all possible combinations of session, task, acquisition and run identifiers).

WDYT @chrisfilo, @shoshber ?

**Comment by chrisfilo** [comment]
LGTM

**Comment by oesteban** [comment]
Fixed in #113
